backend/evals/scannability_qreader.py

```diff
@@ -0,0 +1,46 @@
+import modal
+
+app = modal.App(name="qart-qreader")
+
+
+def download_model():
+    from qrdet import QRDetector
+
+    QRDetector()  # Specify model size here to download non-default ('s' - small)
+
+
+image = (
+    modal.Image.debian_slim()
+    .apt_install("python3-opencv", "libzbar0")
+    .pip_install("opencv-python", "pillow", "pyzbar", "qrdet", "qreader")
+    .run_function(download_model)
+)
+
+
+@app.cls(image=image, gpu="any", allow_concurrent_inputs=10)
```

Comment:
Have we checked whether the GPU is buying us anything here? It's a small YOLO model, so I doubt it.

We might be better off with more cores (say `cpu=2` or`cpu=4`). With `cpu=4`, we'd have a CPU limit of eight cores, and we can set `allow_concurrent_inputs` to `8` so that there are no more concurrent inputs than threads.

That's not scientific, but probably a better setting,